pallets/author-inherent/src/exec.rs

```diff
@@ -57,34 +57,26 @@ where
 		debug!(target: "executive", "In hacked Executive. digests after stripping {:?}", header.digest());
 		debug!(target: "executive", "The seal we got {:?}", seal);
 
-		// let sig = match seal {
-		// 	DigestItem::Seal(id, ref sig) if id == NIMBUS_ENGINE_ID => sig.clone(),
-		// 	_ => panic!("HeaderUnsealed"),
-		// };
 		let signature = seal.as_nimbus_seal().unwrap_or_else(||panic!("HeaderUnsealed"));
 
 		debug!(target: "executive", "🪲 Header hash after popping digest {:?}", header.hash());
 
 		debug!(target: "executive", "🪲 Signature according to executive is {:?}", signature);
 
-		// Grab the digest from the runtime
-		//TODO use the CompatibleDigest trait. Maybe this code should move to the trait.
-		let consensus_digest = header
+		// Grab the author information from either the preruntime digest or the consensus digest
+		//TODO use the trait
```

Comment:
is this TODO outstanding?

Reply:
Yes, it is still outstanding in the sense that it still needs to be done. But I don't want to tackle it in this PR because this is already getting big enough.